### DISCUSSION TITLE
Anybody else has "power not recognised" issues

### DISCUSSION THREAD

**Discussion by ryders** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#771] [General]
Hi folks.. I've only just started using AlDente a few months ago. I'be been having issues where at times, plugging in the power is not recognised -- at al -- by the machine.

Irrespective of battery level, I plug the power and there's no "connected chime", no battery indicator of a "connection to power detected" (plug or bolt), but AlDente is detecting that power is connected (shows a "pause sign"). When that happens, trying to "top up" or change my charge limit up doesn't work to somehow trigger the charge to kick off -- doesn't work. Closing down AlDente doesn't work either, and it seems only rebooting the machine does the trick.

1 - I'm not suggesting this is an AlDente problem, but I have noticed this behaviour since starting to use it
2 - Apple ran a full hardware test and cannot find any hw issue with all ports
3 - has anyone else noticed this or am I going completely insane?

**Comment by MatthiasKerbl** [comment]
Hi @ryders,

I have not heard about this issue before.

Did you try it with different chargers?
Do you encounter this problem when you completely uninstall AlDente?

Cheers,
Matthias

**Reply by ryders** [reply]
Hi @MatthiasKerbl 

I did uninstall+reinstall Al-Dente (even the whole OS). I also thought the charger would be the culprit (I have different chargers lying around the house) but even sticking to 100% genuine 85w Apple charger.. I still get the problem.

The two variables I have isolated this to are:
 
(a) Al-Dente not able to remove a "charge restriction" correctly and preventing charge. Shutting down Al-Dente doesn't work however, i need a full reboot.

(b) I sometimes use a usb-c dongle for monitor+webcam+charge and I've read numerous 'usb-c dongles' issues with sleeping and charging with an M1-chip. So is my dongle a bit doggy? This said, disconnecting the dongle completely, and plugging the original charger directly in, still doesn't resolve the issue.

**Reply by ryders** [reply]
@MatthiasKerbl if I wanted to investigate this further, mostly out of curiosity, is there a way for me to access the effective Al-Dente settings in the OS?

Let me explain.. there are a bunch of options in macos and checkboxes everywhere affecting power/battery usage.. but when I open a terminal and hit `pmset -g` or `pmset -g custom` I see the settings applied in the OS.

Can I somehow see / probe the OS for settings applied by AlDente at all?

**Comment by ryders** [comment]
I've not had the issue in the last 6 weeks. I was not able to clearly identify the pattern to replicate. Closing for now. Will re-open as required.

**Reply by ryders** [reply]
Oh, sorry, this is not an Issue that can be closed, but just a conversation that will live forever. Or so it seems 😆
